events/message/afk-timer.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    name: 'afk-timer',
+    type: 'non-command',
+    aliases: ['example'],
+    description: 'Begins an away timer when specific strings are detected within a message. Stops timer and posts result when the same user posts another message',
+    guildOnly: false,
```

Comment:
I don't think we currently do any ``guildOnly`` checking for non-commands, note to look into this.